Representable/representable
# Variety of polish fixes

**changes**
there are a lot of small ones but putting the biggest ones here for documentation
- only shows comm info sections that were filled out on submission page
- only shows comm info sections that were filled out under "My community info" portion of entry mapping page
- adds another "get started" button further down on the home page
- moves "change mapping units" button on mobile to be on the lower left corner
- fixes some scroll issues with mobile mapping
- makes FAQ font smaller on mobile
- adds more letter spacing for all all-caps buttons on site
- fixes alignment of blue rectangle on home page a bit more
- changes link to current community maps on state page to open a new tab
- changes coloring of text on FAQ and comm info dropdowns
- fixes translation tags showing on password reset emails
- added youtube link to home page + changed alignment
- removed compass from navigation controls on map

**to test**
- python manage.py runserver
- go to each page mentioned above and check for css issues / glitches!
